src/backend/api/upload/add-resume.js

```diff
@@ -0,0 +1,112 @@
+// taken from
+//https://www.labnol.org/google-drive-api-upload-220412
+import { assert } from 'console';
+import { JWT } from 'google-auth-library';
+import {google} from 'googleapis';
+import { objOf } from 'ramda';
+import stream from 'stream';
+
+// get auth using the correct service account
+const getAuth = async () => {
+  return new JWT({
+        scopes:[
+          "https://www.googleapis.com/auth/drive.file",
+          "https://www.googleapis.com/auth/drive",
+          "https://www.googleapis.com/auth/drive.appdata",
+          "https://www.googleapis.com/auth/drive.metadata",
+        ],
+        email:'teamhubbackend@teamhub-257722.iam.gserviceaccount.com',
+        key: process.env.SERVICE_ACCOUNT_PRIVATE_KEY,
+    })
+}
+
+// create the folder if it does not exist
+const putFolder = async (folderName, parentFolder, jwtClient) => {
```

Comment:
Add a method doc:

`/**
   *
   */
`

Defining the return value and input params.